### ISSUE TITLE
jobhelper.xsl should use XSLT 2.0

### ISSUE THREAD

**Issue by xephon2** [dita-ot/dita-ot#2831]
The **jobhelper.xsl** (`dita-ot\xsl\job-helper.xsl`) uses plain old XSLT 1.0:

```xml
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
```

As I understand [Michael K.'s comment](https://stackoverflow.com/a/45600242/3604444), Saxon HE does not support this anymore. At least it crashes [my build](https://travis-ci.org/doctales/org.doctales.schematron/builds/296179023#L585).

**DITA-OT**: `2.5.2`

@infotexture, the **messages.xsl** (`dita-ot\docsrc\resources\messages.xsl`) also uses XSL 1.0.

**Comment by infotexture** [comment]
@xephon2 Both of these files already use XSLT 2.0:

* The [job-helper.xsl](https://github.com/dita-ot/dita-ot/blob/develop/src/main/xsl/job-helper.xsl#L10) file was updated to XSLT 2.0 on June 12 in cbde40d36.
* The [messages.xsl](https://github.com/dita-ot/docs/blob/develop/resources/messages.xsl#L4) file was updated to XSLT 2.0 on June 30 in dita-ot/docs@1c3f5868.

**Comment by xephon2** [comment]
@infotexture, that has been fixed for **DITA-OT** 3.0, but maybe should be fixed for 2.5.4, or is the 2.x now finished (due to missing resources)?

**Comment by robander** [comment]
@xephon2 I do expect at least one or two 2.5.x patches, so this could go in, but I'm not sure whether it's worth it given that the code is working in those releases with the version of Saxon those include? 

(Definitely don't want to start the clock on a patch with just this, but I don't have strong feelings about adding this in once we've already got 2.5.5 planned.)
